src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -1054,7 +1054,7 @@ private CustomAttributesBag<CSharpAttributeData> GetAttributesBag(ref CustomAttr
         /// Gets the attributes applied on this symbol.
         /// Returns an empty array if there are no attributes.
         /// </summary>
-        public sealed override ImmutableArray<CSharpAttributeData> GetAttributes()
+        public override ImmutableArray<CSharpAttributeData> GetAttributes()
```

Comment:
Don't we already have some constructs that need to copy over attributes? How do they hand out attributes?

Reply:
I don't think we would want to override ```GetAttributeDeclarations``` for the purpose of this PR. It returns syntax for binding attributes. We don't want to re-bind attributes, just want to pretend we have the same attributes as the ```BaseMethod```. Overriding ```GetAttributes``` looks like a reasonable approach.  

---
In reply to: [338276323](https://github.com/dotnet/roslyn/pull/39226#discussion_r338276323) [](ancestors = 338276323)